examples/jpa-hibernate-arquillian/README.adoc

```diff
@@ -0,0 +1,323 @@
+= JPA Hibernate Arquillian
+:index-group: JPA
+:jbake-type: page
+:jbake-status: published
+
+This example shows the persist, remove and creation a query in JPA Hibernate 5 using arquillian for the test.
```

Comment:
This example shows the persist, remove and creation **of an entity** [...]

Reply:
Thank you, also fixed in commit ea49d8a9bf